pkg/converter/testdata/docker-compose-v3-simple.yml

```diff
@@ -0,0 +1,57 @@
+version: "3"
+
+services:
+  busy_env:
+    image: busybox
+    env_file: testdata/test.env
+    command: ["sleep", "20"]
+    depends_on:
+      - wordpress
+    links:
+      - db
+  wordpress:
+   domainname: mywords.sloppy.zone
+   image: wordpress:4.7.4
+   ports:
+     - "8000:80"
+   restart: always
+   environment:
+     WORDPRESS_DB_HOST: db:3306
+     WORDPRESS_DB_USER: wordpress
+     WORDPRESS_DB_PASSWORD: wordpress
+   entrypoint: ["/code/entrypoint.sh", "-foo", "bar"]
+   deploy:
+     mode: replicated
+     replicas: 3
```

Comment:
Shouldn't this and the memory properties below get mapped to `instances` and `mem`?

May need a 2nd test case (non-simple) to be able to compare this to the v2 input.